### ISSUE TITLE
retrun

### ISSUE THREAD

**Issue by 100petr** [nulli0n/GoldenCrates#66]
Probably an bug: If I'm on page 3 with rewards, I click on an item to edit it, then retrun and I'm back on the first page.
GC 3.11.0

**Comment by nulli0n** [comment]
Yes, I'll see if can I fix it (some issues in the engine's GUI handler)

**Comment by 100petr** [comment]
lore:
    - '&d[&7Zobrazení vyžaduje OptiFine&d]'

**Comment by 100petr** [comment]
something new? I would like to add about 130 items to the chest, but this way it will be nerve-wracking
